25_exceptions_lab/01.py

```diff
@@ -1,7 +1,17 @@
-"""
-Write a python program that takes numbers in a loop
-and for each number prints its square root
-If value is negative or not a number show 
-a warning and keep reading values
-"""
-
+import math
+def sqrt(n):
+    if n < 0:
+        raise ValueError("Expected parameter >= 0, Got: %d" % n)
+    return math.sqrt(n)

```

Comment:
No need to write your own `sqrt` function - python already has one and it's already raising the correct exceptions.